surveys/promise-rejections/survey.md

```diff
@@ -0,0 +1,68 @@
+# Node.js Promise reject use case survey
+
+## Are you currently using Promises, async/await, a mix, or neither?
+
+Assume using as creating your own `new Promise` and `async function`
+
+ - [ ] Promises
+ - [ ] async/await
+ - [ ] Both
+ - [ ] Neither / not writing Promise-based code
+
+## Are you currently using `.catch()`, `try await catch`, a mix, or neither to handle rejections?
+
+When consuming Promises, async functions or thenables, which of the options below do you use to handle rejections?
+
+ - [ ] `.catch()`
+ - [ ] `try await catch`
+ - [ ] Both
+ - [ ] Neither / not writing Promise-based code
```

Comment:
I would add an other option here. Some codebases rely on a mixture of strategies and it's often inconsistent when there are multiple teams involved so this can be a difficult question to ask.

`Ignore the rejection` should likely also be an explicit option here.